Cargo.toml

```diff
@@ -2,21 +2,26 @@
 description = "rust port of identicon generator for polkadot from polkadot.js"
 license = "Apache-2.0"
 name = "plot_icon"
-version = "0.1.3"
-authors = ["Alexander Slesarev <slesarew@gmail.com>"]
+version = "0.1.4"
+authors = ["Alexander Slesarev <slesarew@gmail.com>", "Vera Abramova <abramova.vera@gmail.com>"]
 edition = "2021"
 repository = "https://github.com/paritytech/polkadot-identicon-rust"
 homepage = "https://github.com/paritytech/polkadot-identicon-rust"
 keywords = ["identicon", "icon", "parity", "polkadot", "substrate"]
 
 [dependencies]
-hex = "0.4.3"
-png = "0.16.8"
 blake2-rfc = "0.2.18"
-palette = "0.6.0"
-base58 = "0.1.0"
-svg = "0.10.0"
-anyhow = "1.0.42"
+hex = {version = "0.4.3", optional = true}
+image = {version = "0.24.0", default-features = false, features = ["ico"], optional = true}
+palette = {version = "0.6.0", default-features = false, features = ["std"]}
+png = {version = "0.17.3", optional = true}
+svg = {version = "0.10.0", optional = true}
+
+[features]
+default = ["pix", "vec"]
+
+pix = ["image", "png"]
+vec = ["hex", "svg"]
```

Comment:
nit: the `feature` naming needs some additional attention :bike: :house:

Reply:
Left as it was for now. They are somewhat described in readme and docs.